test/ranges-test.cc

```diff
@@ -11,17 +11,18 @@
 
 #include "fmt/ranges.h"
 
-#include "gtest.h"
+#include <array>
+#include <map>
+#include <string>
+#include <vector>
 
-// Check if  'if constexpr' is supported.
-#if (__cplusplus > 201402L) || \
-    (defined(_MSVC_LANG) && _MSVC_LANG > 201402L && _MSC_VER >= 1910)
+#include "gtest.h"
 
-#  include <array>
-#  include <map>
-#  include <string>
-#  include <vector>
+#if FMT_GCC_VERSION == 0 || FMT_GCC_VERSION >= 601
+#  define FMT_ENABLE_C_STYLE_ARRAY_TEST
+#endif
```

Comment:
It's strange but looks like GCC <6 unable to pass C-style array correctly, so {fmt} is unable right now to format them - https://godbolt.org/z/3jv87x